### DISCUSSION TITLE
Ditching GitHub due to algorithmic social media feed

### DISCUSSION THREAD

**Discussion by chakravala** [community/community#66769] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Due to GitHub removing the old feed, I will stop using GitHub and I switch to alternatives.

All I want is the simple old feed, the new look is awful.

I've deleted all my social media accounts, probably need to delete my GitHub account as well, I don't want social media.

**Comment by SpriteOvO** [comment]
[GitLab](GitLab.com) is an alternative option, or if you want to self-host, check [Gitea](Gitea.com).

**Reply by Zaczero** [reply]
The community has forked Gitea into [Forgejo](https://forgejo.org/) now. Gitea is currently controlled by a for-profit company, and I personally recommend avoiding it. More details in the [FAQ](https://forgejo.org/faq/). Also, check out https://codeberg.org for a free managed instance.
